microsoftgraph/msgraph-sdk-python-core
# Add workflows to automate package publishing

## Overview

Adds workflows to automate creating a new release as well as publishing to PyPI

### Demo

![image](https://user-images.githubusercontent.com/40166690/125908764-e295fbf0-d1a3-4a1a-99cd-5f018b045228.png)

## Testing Instructions

* Tag a commit with a new version of the package and push to github
* Observe that the release workflow is triggered and automatically drafts a new release
* Review the draft release and make the necessary adjustments
* On publishing the release, the publish workflow automatically publishes the new version to PyPI

Fixes #56 
Fixes #31
